Defra.Cdp.Backend.Api.IntegrationTests/Services/Deployments/DeploymentServiceTest.cs

```diff
@@ -0,0 +1,196 @@
+using Defra.Cdp.Backend.Api.IntegrationTests.Mongo;
+using Defra.Cdp.Backend.Api.Models;
+using Defra.Cdp.Backend.Api.Mongo;
+using Defra.Cdp.Backend.Api.Services.Aws.Deployments;
+using Defra.Cdp.Backend.Api.Services.Deployments;
+using Defra.Cdp.Backend.Api.Services.Github;
+using Defra.Cdp.Backend.Api.Services.Github.ScheduledTasks;
+using Microsoft.Extensions.Logging.Abstractions;
+using NSubstitute;
+using NSubstitute.ReturnsExtensions;
+
+namespace Defra.Cdp.Backend.Api.IntegrationTests.Services.Deployments;
+
+public class DeploymentServiceTest : IClassFixture<MongoIntegrationTest>
```

Comment:
We have `ServiceTest` which will set up & tear down the database

Reply:
i've moved this out into Utils so its a bit easier to discover